index.js

```diff
@@ -796,6 +796,9 @@
   Nothing.prototype.toString = def('Nothing#toString', [],
                                    R.always('Nothing()'));
 
+  //  Nothing#inspect :: Maybe a ~> String
+  Nothing.prototype.inspect = R.nAry(0, Nothing.prototype.toString);
+
```

Comment:
* `inspect` will be provided with two arguments when called by the REPL and `toString` is nullary so we do need to control the arguments in some fashion.

* Agreed, it makes more sense attached to `Maybe`, :zap:ing.